oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/ProfileMutation.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.skywalking.oap.query.graphql.resolver;
+
+import com.coxautodev.graphql.tools.GraphQLMutationResolver;
+import org.apache.skywalking.oap.query.graphql.type.ProfileTaskCreationRequest;
+import org.apache.skywalking.oap.server.core.CoreModule;
+import org.apache.skywalking.oap.server.core.profile.ProfileTaskMutationService;
+import org.apache.skywalking.oap.server.core.profile.entity.ProfileTaskCreationResult;
+import org.apache.skywalking.oap.server.core.query.DurationUtils;
+import org.apache.skywalking.oap.server.library.module.ModuleManager;
+
+import java.io.IOException;
+
+/**
+ * profile mutation GraphQL resolver
+ *
+ * @author MrPro
+ */
+public class ProfileMutation implements GraphQLMutationResolver {
+
+    private final ModuleManager moduleManager;
+    private ProfileTaskMutationService profileTaskService;
+
+    public ProfileMutation(ModuleManager moduleManager) {
+        this.moduleManager = moduleManager;
+    }
+
+    private ProfileTaskMutationService getProfileTaskService() {
+        if (profileTaskService == null) {
+            this.profileTaskService = moduleManager.find(CoreModule.NAME).provider().getService(ProfileTaskMutationService.class);
+        }
+        return profileTaskService;
+    }
+
+    public ProfileTaskCreationResult createProfileTask(ProfileTaskCreationRequest creationRequest) throws IOException {
+        return getProfileTaskService().createTask(
+                creationRequest.getServiceId(),
+                creationRequest.getEndpointName(),
+                creationRequest.getStartTime() == null ? -1 : creationRequest.getStartTime(),
+                Math.toIntExact(DurationUtils.INSTANCE.toSecond(creationRequest.getDurationUnit(), creationRequest.getDuration())),
```

Comment:
Reading this makes me feel, we should set the duration in minute level only, and remove the step in the protocol. @kezhenxu94 @arugal  What do you think?

Reply:
> Reading this makes me feel, we should set the duration in minute level only, and remove the step in the protocol. @kezhenxu94 @arugal What do you think?

Make sense to me, but `minute` level seems too large in time span, what about making it `second` level?

Reply:
Yes, agree. Let's change this is `ms` level, but to be clear, this is not for span, it is for segment

Reply:
According to https://github.com/apache/skywalking-query-protocol/blob/master/profile.graphqls#L25-L26, this duration is for the task, I think minute makes more sense.

Reply:
This threshold for segment(nothing related to span) is minDurationThreshold, refer to https://github.com/apache/skywalking-query-protocol/blob/master/profile.graphqls#L30